.github/ISSUE_TEMPLATE/update-dependencies.md

```diff
@@ -8,11 +8,13 @@ assignees: ''
 
 ## 📚 Context
 
-On-chain or Off-chain?
+Which dependencies should be bumped, and to which versions?
+Are there any clear benefits? (new features were added, bugs were fixed, etc)
+Are there any clear downsides? (requires refactoring, bugs were introduced, etc)
```

Comment:
Are they meant to be in separate lines? They seem to be displayed in the same line eventually.

Reply:
I mean I'm not sure if this is what you expected
```
Which dependencies should be bumped, and to which versions? Are there any clear benefits? (new features were added, bugs were fixed, etc) Are there any clear downsides? (requires refactoring, bugs were introduced, etc)
```
instead of 
```
Which dependencies should be bumped, and to which versions?
Are there any clear benefits? (new features were added, bugs were fixed, etc)
Are there any clear downsides? (requires refactoring, bugs were introduced, etc)
```
If you want the latter, you'll need extra empty lines between them :)

Reply:
When you create an issue from a template, it is on "source" mode, so lines are separated like in the second code block.